intelmq/bots/outputs/restapi/output.py

```diff
@@ -34,7 +34,13 @@ def process(self):
                               cert=self.ssl_client_cert,
                               timeout=self.http_timeout_sec,
                               **kwargs)
-        r.raise_for_status()
+        if self.parameters.logging_level in ["DEBUG", 10] and r.status_code != 200:
+            try:
+                self.logger.debug("Something failed during message sending with response body {}".format(r.text))
```

Comment:
Please let the logging library do the formatting (avoids bad error message when formatting fails), e.g.:
```python
self.logger.debug("Received response body: %r.", r.text)
```